articles/tales-from-fleet-security-securing-1password.md

```diff
@@ -0,0 +1,81 @@
+# Tales from Fleet security: securing 1Password
+
+![Securing 1Password](http://fleetdm.com/images/articles/securing-1password-1600x900@2x)
+
+No matter how much we wish SaaS tools had support for Single Sign-On (SSO), there are still so many
+websites and other resources that require individual passwords. Running a company without providing
+employees with a password manager is downright dangerous.
+
+At Fleet, we use 1Password. While configuring 1Password is relatively straightforward, here are a few things we do that can help you secure your 1P instance.
+
+## Enforce 2FA
+By setting the Account Password Policy to Strong, we gained the ability to support and then enforce 2FA.
+
+When enabling 2FA, we ensured we also enforced it, as otherwise, there would always be some users without it.
```

Comment:
How did we enforce it? Could we add a short line? e.g.: 

> "By setting the Account Password Policy to Strong, we gained the ability to support 2FA. We then enforced it by [insert means here], to ensure that the whole team adopted the same approach."

Also, and this is more of a musing, I wonder if there's a more positive 
synonym for "Enforce." It's a typical security word choice, but it always feels so... forceful.

Reply:
I think *required* is slightly better?